case_studies/sdss_galaxies/plots.py

```diff
@@ -142,38 +143,34 @@ def fignames(self):
             "classification": "sdss-classification-acc.png",
         }
 
-    def compute_data(self, scene, coadd_cat, sleep_net, binary_encoder, galaxy_encoder):
+    def compute_data(self, scene, coadd_cat, encoder, decoder):
         assert isinstance(scene, (torch.Tensor, np.ndarray))
-        assert sleep_net.device == binary_encoder.device == galaxy_encoder.device
-        device = sleep_net.device
+        assert encoder.device == decoder.device
+        device = encoder.device
 
         bp = 24
-        clen = 300
+        slen = 300  # chunk side-length
         h, w = scene.shape[-2], scene.shape[-1]
 
```

Comment:
I don't think it will be a hard change to handle rectangular images.

Reply:
I can push some changes to this branch to allow it so we don't have to have this "FIXME" here.

Reply:
I merged in #433 so this can be fixed